src/modals/InfoModal.css

```diff
@@ -52,6 +52,5 @@ ion-button.ok-modal {
 }
 
```

Comment:
This is the hover style. It does not change, so it can be removed.
The properties above (--color and --background) are also not needed.

Reply:
Removed